reactioncommerce/reaction-admin
# Feature: use new Catalyst data table in products view and update orders view

Resolves #181  
Impact: major  
Type: refactor

## Summary
This PR integrates the new `DataTable` from the Reaction Catalyst design system in the products view. As a side effect, many Meteor method dependencies will be removed.

## Breaking changes
None


## Testing
1. Verify all bulk actions in the products view work as intended, paying attention to success/error notifications for correctness.
2. This PR also adds a couple of filters to the orders table, verify all filters work as intended.
3. Verify all unnecessary Meteor publications have been removed.
